tests/sentry/integrations/slack/test_notify_action.py

```diff
@@ -118,7 +118,7 @@ def test_upgrade_notice_over_threshold(self, mock_uniform):
         mock_uniform.assert_called_with(0, 1)
 
     @responses.activate
-    @patch("random.uniform", return_value=0.049)
+    @patch("random.uniform", return_value=0.51)
```

Comment:
@MeredithAnya Shouldn't this be <0.5 for this test to pass?

Reply:
I thought this test was to prove that it shouldn't send for bot apps

Reply:
oh but it still needs to be under threshold huh

Reply:
@MeredithAnya Well, we send the message for workspace apps when the randomly generated value is less than `UPGRADE_MESSAGE_FREQUENCY`. So if you made this test for a workspace app, it would still pass.